frontend/src/pages/RecipePage.js

```diff
@@ -22,7 +22,7 @@ export default function RecipePage() {
                 <img src={recipe.imageUrl} alt={recipe.name}/>
                 <RecipeHeader recipeName={recipe.name} recipeDescription={recipe.diaryEntry}/>
                 <RecipeCoronaDetails quarantineDay={recipe.quarantineDay}/>
-                <RecipePreparation ingredients={recipe.ingredients} preparationSteps={recipe.steps}/>
+                <RecipePreparation ingredients={recipe.ingredients} preparationSteps={recipe.steps} amountOfPeople={recipe.amountOfPeople}/>
```

Comment:
You could simplify this by renaming the `preparationSteps` prop to just `steps` and then do
```suggestion
                <RecipePreparation {...recipe}/>
```
Spread operator is your friend when object keys and prop names match ;)

Reply:
yes great point, thx for pointing that out 👍🏻